openreview/venue/process/comment_pre_process.js

```diff
@@ -0,0 +1,24 @@
+async function process(client, edit, invitation) {
+  client.throwErrors = true
+
+  const { note } = edit
+  
+  if (note.ddate || note.readers === ['everyone']) {
```

Comment:
i think the everyone comparison logic should be changed to use includes
current logic will always be false

Reply:
like this? `note.readers.includes('everyone')`?

Reply:
yes
or readers[0]==='everyone'

Reply:
fixed!